src/components/Account-panel/Change-password/Change-password.tsx

```diff
@@ -41,97 +25,74 @@ export const ChangePassword = () => {
   } = useForm<FormData>({
     resolver: yupResolver(ChangePasswordSchema)
   })
-  const [snackbarData, setSnackbarData] = useState<SnackbarData>({
-    open: false,
-    error: false
-  })
+  const { showSnackbar } = useSnackbar()
+
   const onSubmit = async (data: FormData) => {
     setIsLoadingBtn(true)
     try {
       const response = await axiosInstance.post(
         '/api/users/me/change_password',
         data
       )
-      setSnackbarData({ open: true, error: false, message: 'Пароль змінено!' })
+      showSuccessSnackbar()
       return response
     } catch (error) {
-      setSnackbarData({
-        open: true,
-        error: true,
-        message: 'Сталась помилка, спробуйте ще раз...'
-      })
       console.error('Error updating user data:', error)
+      showErrorSnackbar()
     } finally {
       setIsLoadingBtn(false)
       reset()
     }
   }
 
-  const handleCloseSnackbar = () => {
-    setSnackbarData({ ...snackbarData, open: false })
+  const showSuccessSnackbar = () => {
+    showSnackbar({ error: false, message: 'Пароль змінено!' })
+  }
+
+  const showErrorSnackbar = () => {
+    showSnackbar({
+      error: true,
+      message: 'Сталась помилка, спробуйте ще раз...'
+    })
```

Comment:
you don't need another functions if you use it once

Reply:
done